README.md

```
@@ -1,174 +1,306 @@
+
+<div align="center">
+  <a href="https://roamance.onrender.com/api/swagger-ui/index.html">
+    <img src="https://img.shields.io/badge/API-Documentation-007ACC?logo=swagger&logoColor=white&style=for-the-badge" alt="API Documentation" />
+  </a>
+  <a href="#-getting-started">
+    <img src="https://img.shields.io/badge/Getting%20Started-Guide-4CAF50?logo=spring&logoColor=white&style=for-the-badge" alt="Getting Started Guide" />
+  </a>
+</div>
+
+---
+## 📋 Table of Contents
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)